.yamato/hdrp-linux-vulkan.yml

```diff
@@ -78,17 +78,17 @@ HDRP_Linux_Vulkan_Standalone_mono_apiNet2_Linear_trunk:
     commands:
       -  curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr --output TestProjects/HDRP_RuntimeTests/utr
       -  chmod +x TestProjects/HDRP_RuntimeTests/utr
-      -  cd TestProjects/HDRP_RuntimeTests && ./utr --artifacts_path=test-results --extra-editor-arg="-colorspace=Linear" --platform=StandaloneLinux64 --player-connection-ip=auto --player-load-path=../../players --reruncount=2 --scripting-backend=Mono2x --suite=playmode --timeout=1200
```

Comment:
is the removal of colorspace and scripting backend flags intentional?

Reply:
yes, those flags only work for performance tests